examples/ssh/crestron_touch_screen.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Domotz Custom Driver 
+ * Name: Crestron Touch Screen Monitoring 
+ * Description: This drivers monitors Front Panel Slot and Core3UILevel sensors
+ * 
+ * Communication protocol is SSH.
+ * 
+ * Creates a Custom Driver Table with the following colums:
+ *  - Sensor
+ *  - Value
+**/
+
+// Ssh options and command to be run
+var command ="info"
+var options = {
+    "command": command,
+    "username": D.device.username(),
+    "password": D.device.password(),
+    "timeout": 35000
+};
+
+// Helper function to parse the df response and call the success callback
+function successCallback(output) {
+    // Creation of custom driver table 
+    var table = D.createTable(
+        "Touchscreen Sensors",
+        [
+            { label: "Sensor" },
+            { label: "Value" }
+        ]
+    );
+
+var outputArr = output.split(/\r?\n/);
+var outputArrLen = outputArr.length;
+
+for (var i = 0; i < outputArrLen; i++) {
+    var fields = outputArr[i].replace(/\s+/g,' ').trim();
+    var uidN = i;
+    var uid=uidN.toString();
```

Comment:
this is technically the record id

better change it to 
var recordId = i.toString()

Although it feels dangerous for the index to be the unique identifier considering their order may change and it may have other entries to offset the whole thing

better use a known string eg

var recordId = 'front-panel-slot'